src/components/app/app.ts

```diff
@@ -1,17 +1,25 @@
 import AppController from '../controller/controller';
 import { AppView } from '../view/appView';
+import { IDataSources } from '../view/sources/sources';
 
 class App {
+    controller: AppController;
+    view: AppView;
```

Comment:
Making the fields of a class public unless strictly necessary is not good.
Please make them private.

Reply:
Done.